subiquity/tests/api/test_api.py

```diff
@@ -847,6 +850,68 @@ async def test_use_free_space_after_existing(self):
             }
             self.assertDictSubset(e4, p4)
 
+    @timeout()
+    async def test_resize(self):
+        # load config, edit size, use that for server
+        with open('examples/ubuntu-and-free-space.json', 'r') as fp:
+            data = json.load(fp)
+        # expand to use the rest of the disk
+        data['storage']['blockdev']['/dev/sda3']['attrs']['size'] = 40899706880
```

Comment:
This number is a mystery guest. Can you read the disk size out of the blockdev data for /dev/sda or something?

Reply:
Fixed in the rebase.